policyengine_us/variables/gov/states/nm/tax/income/credits/nm_child_income_credit.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class nm_child_income_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Mexico child income tax credit"
+    definition_period = YEAR
+    reference = "https://nmonesource.com/nmos/nmsa/en/item/4340/index.do#!fragment/zoupio-_Toc140503818/BQCwhgziBcwMYgK4DsDWszIQewE4BUBTADwBdoAvbRABwEtsBaAfX2zgEYAWABgFYeAZgAcHYQEoANMmylCEAIqJCuAJ7QA5BskRCYXAiUr1WnXoMgAynlIAhdQCUAogBknANQCCAOQDCTyVIwACNoUnZxcSA"
+    defined_for = StateCode.NM
+
+    def formula(tax_unit, period, parameters):
+        income = tax_unit("nm_agi", period)
```

Comment:
The law refers only to `Adjusted gross income`, does that mean NM AGI or federal?